### PR TITLE
(tlg0018-cohn2) updates to the Cohn/Wendland Greek of Philo

### PR DESCRIPTION
**Pull request by gregorycrane**
explanations are here: https://github.com/OpenGreekAndLatin/First1KGreek/issues/2754

### CODE REVIEW

File: data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -94,65 +94,61 @@
   </langUsage>
  </profileDesc>
  <revisionDesc>
-  <change who="GRC" when="2023-05-24">GRC added 169 corrections, each marked with a <corr/> tag.</change>
```
Comment (lcerrato) [inline data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml:97]: I would keep this note as the changes are extensive enough to warrant it. Not jus the tag removal but also the xml:base addition

File: data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml [hunk 2]
```diff
@@ -161,205 +157,214 @@ Philonis opera vol. I 1</note>
...
-<lb n="17"/> συνεκτικοτάτων τῆς φύσεως V 18 δή, ὅτι] διότι VM Eus δραστήριον ὡς
-αἴτιον MABP 19 αἴτιον—δραστήριον om. Eus 20 κρείττων ἡ ἀρετὴ V, κρείττω
-τε ἡ ἀρετὴ M, κρεῖττον τε ἡ ἀρετὴ AB, κρείττων τε ἡ ἀρετὴ ceteri, κρείττων ἢ ἀρετὴ
-ω
-καὶ om. Eus καὶ κρεῖττον M, καὶ κρεῖττον AB</note>
-<note type="footnote">6—11 ὡς—ὑποσιωπῆσαι N fol. 160v 12—3,19 Τινὲς—θεολογήσας Euseb.
-VIII 13</note>
-
-<pb n="3"/>
+<note xml:lang="mul" type="footnote">
```
Comment (lcerrato) [inline data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml:173]: NB I do not find that `<lb/>` works in notes. `<p>` does

File: data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml [hunk 3]
```diff
@@ -368,186 +373,187 @@ V: εἰ ceteri (H in ras.) συνέστηκεν V: ὑφέστηκεν ceteri 9
 ἐφθόνησεν οὐσίᾳ μηδὲν ἐξ αὑτῆς ἐχούσῃ καλόν, δυναμένῃ δὲ πάντα
  γίνεσθαι.</p></div>
 
-<div type="textpart" subtype="section" n="22">
-<p>ἦν μὲν γὰρ ἐξ αὑτῆς ἄτακτος ἄποιος ἄψυχος <corr>&lt;ἀνόμοιοσ&gt;,</corr>
+<div type="textpart" xml:base="urn:cts:greekLit:tlg0018.tlg001.opp-grc1" subtype="section" n="22">
+<p>ἦν μὲν γὰρ ἐξ αὑτῆς ἄτακτος ἄποιος ἄψυχος &lt;ἀνόμοιοσ&gt;,
```
Comment (lcerrato) [inline data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml:377]: entities should be removed. 〈 〉etc.  can be added here.

File: data/tlg0018/tlg003/tlg0018.tlg003.opp-grc1.xml [hunk 4]
```diff
@@ -94,107 +94,121 @@
...
 <head>ΠΕΡΙ ΤΩΝ ΧΕΡΟΥΒΙΜ
 ΚΑΙ ΤΗΣ ΦΛΟΓΙΝΗΣ ΡΟΜΦΑΙΑΣ
 KAI ΤΟΥ ΚΤΙΣΘΕΝΤΟΣ ΠΡΩΤΟΥ ΕΞ ΑΝΘΡΩΠΟΥ ΚΑΙΝ</head>
-<div type="textpart" subtype="section" n="1">
-<p><milestone unit="altref" n="1"/> „Καὶ ἐξέβαλε τὸν Ἀδὰμ καὶ κατῴκισεν ἀπέναντι τοῦ παραδείσου <note type="marginal">p. 138 M.</note>
+<div type="textpart" xml:base="urn:cts:greekLit:tlg0018.tlg003.opp-grc1" subtype="section" n="1">
+<p><milestone unit="altref" n="1"/> „Καὶ ἐξέβαλε τὸν Ἀδὰμ καὶ κατῴκισεν ἀπέναντι τοῦ παραδείσου <note xml:lang="lat" type="marginal">p. 138 M.</note>
 τῆς τρυφῆς [καὶ ἔταξε] τὰ Χερουβὶμ καὶ τὴν φλογίνην ῥομφαίαν
 τὴν στρεφομένην, φυλάσσειν τὴν ὁδὸν τοῦ ξύλου τῆς ζωῆς“ (Gen. 3, 24).
 νῦν μὲν „ἐξέβαλε“ φησί, πρότερον δ’ εἶπεν „ἐξαπέστειλεν“ (ib. 23),
```
Comment (lcerrato) [inline data/tlg0018/tlg003/tlg0018.tlg003.opp-grc1.xml:112]: `<q>` tags generally substituted here

**Review by lcerrato — COMMENTED** [review]
I have only done a spot check due to the enormous size and the inability to efficiently scroll through all of the files. A few suggestions:

- the change log was removed, but I think it belongs here due to the extensive nature of the changes, esp if the changes were automated
- `<lb/>` in notes doesn't seem to do much. I presume this is a stopgap until notes are placed inline (or note cross refs are refined?) but I use `<p>` as that works in the viewer.
- there are entities lurking here
- there are quotes here

None of this should prevent ingestion as the file is certainly improved from current version, but these are all things that are part of the usual refinement workflow

File: data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml (hunk 1 above)
Comment (lcerrato) [reply data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml:97]: in the case where stuff was removed or added automatically, this is even more important to alert future editors, I think

**Comment by gregorycrane** [comment]
By popular demand, &lt; and &gt; entities have been removed and replaced with \<add/> tags -- it took some time because there were missing angle brackets and these popped up with unbalanced \<add/> tags. But it is done.

**Comment by gregorycrane** [comment]
Oh- and I put back the \<change/> statements such as:

  <change who="Gregory Crane" when="2023-08-01">Original text was generated with OCR in 2013 and many errors had crept in. GRC, through manual and automated methods, made more than 123 corrections. Markup was also refined and changed throughout.</change>

**Review by lcerrato — COMMENTED** [review]
cts Greek edition labels should be in Greek (can be modified later if flagged)

HathiTrust links include specific view modifications

**Comment by gregorycrane** [comment]
I have made all my changes. Should we ingest this?

**Comment by lcerrato** [comment]
Greek and Latin language declarations were missing from the headers of the translation files (not the cts metadata) where `<foreign>` appeared in the body of the translation. So these languages were tagged in the text but never declared in the headers.

Sometimes the links to the pdfs have things like a zoom or other modifications in them. I think I eliminated the instance where I saw this but it looks like the creators used 1up instead of 2up, which is our usual practice. That can be fixed globally.

**Comment by gregorycrane** [comment]
Do you need to approve the PR ? Merging is still blocked.

**Comment by lcerrato** [comment]
I haven’t reviewed your changes, was working on English

**Review by lcerrato — APPROVED** [review]

**Pull request merged**